src/metatrain/experimental/gap/schema-hypers.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "enum": ["experimental.gap"]
+    },
+    "model": {
+      "type": "object",
+      "properties": {
+        "soap": {
```

Comment:
it would be interesting to check if we can automatically import rascaline's JSON schema file here.

Reply:
Might be very cool. But then we have to generate this schemas dynamically based on the location of rascaline or what do have in mind?

Reply:
rascaline currently does not distribute schema (they are only used for documentation generation). I was thinking we could copy the file from rascaline next to this one, and update it whenever we update the pin on rascaline version

Reply:
Yes, sounds like a good idea. If there is a way to use a distributed schema we can try it once we changed the schema in rascaline.